lib/landrush/util/dnsmasq.rb

```diff
@@ -0,0 +1,10 @@
+module Landrush
+  module Util
+    class Dnsmasq
+      def self.nm_managed?
+        nm_config = Pathname('/etc/NetworkManager/NetworkManager.conf')
+        File.exist?(nm_config) && File.readlines(nm_config).grep(/^dns=dnsmasq$/).any?
```

Comment:
Yes. I believe it has to be enabled manually. It doesn't trigger the dnsmasq service to run as far as I know either; it runs under the NetworkManager service process, not to be confused with the dnsmasq processes that may be run by libvirt in a similar manner.